src/body/mod.rs

```diff
@@ -188,7 +188,7 @@ impl AsyncBody {
 impl AsyncRead for AsyncBody {
     fn poll_read(
         mut self: Pin<&mut Self>,
-        cx: &mut Context<'_>,
+        cx: &mut Context,
```

Comment:
Using an explicit `<'_>` is (or was) preferred here, and is checked by the [`elided_lifetime_in_path`](https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#elided-lifetimes-in-paths) lint. Though while it was part of the `rust_2018_idioms` lint group, it apparently is still not enabled by default and there is no `rust_2021_idioms` lint group.

While the [docs still recommend an explicit lifetime still](https://doc.rust-lang.org/reference/lifetime-elision.html#lifetime-elision-in-functions), apparently there's [still a debate](https://github.com/rust-lang/rust/issues/91639) as to whether it should be recommended or not. Personally I do think it makes it a bit more clear for readers while being more tedious for writers, but generally I am of the opinion of optimizing code for reading (you write it once or twice, but read it hundreds of times).

Reply:
Addressed.